lambdas/service/app.py

```diff
@@ -84,12 +84,66 @@
     'tags': [
         {
             'name': 'Health',
-            'description': 'For checking on service health'
+            'description': 'Describes the health status of Azul service.'
         }
     ],
     'servers': [
         {'url': config.service_endpoint()}
-    ]
+    ],
+    'components': {
```

Comment:
Instead of defining the responses in the global spec object, you could define it in the `health_spec` variable like
```
health_spec = {
    'tags': ['Health'],
    'responses': {
        '200': {
            'description': 'Specified service component is reachable.',
            'content': {
...
```
I think, if we can we should avoid defining responses in `components.responses` and instead rely on Python's reuse functionality (i.e. the variable in this case).